### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'hasOwnProperty') Console error thrown while using GTranslate widget and logged into website as admin

### ISSUE THREAD

**Issue by adamdunnage** [google/site-kit-wp#5969]
## Bug Description

As flagged in the [support forum](https://wordpress.org/support/topic/after-logging-in-the-translation-using-gtranslate-doesnt-work/#post-16070703), the following console error is thrown while using GTranslate plugin widget and logged into website as admin:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')
    at _._setupNS (/_/translate_http/_/js/k=translate_http.tr.cs.sS79CTwCGSs.O/d=1/rs=AN8SPfooco5r-XPZgZHhoMWywin50i9rzg/m=el_conf:261:314)
    at /_/translate_http/_/js/k=translate_http.tr.cs.sS79CTwCGSs.O/d=1/rs=AN8SPfooco5r-XPZgZHhoMWywin50i9rzg/m=el_conf:287:17
    at /_/translate_http/_/js/k=translate_http.tr.cs.sS79CTwCGSs.O/d=1/rs=AN8SPfooco5r-XPZgZHhoMWywin50i9rzg/m=el_conf:292:11
    at /_/translate_http/_/js/k=translate_http.tr.cs.sS79CTwCGSs.O/d=1/rs=AN8SPfooco5r-XPZgZHhoMWywin50i9rzg/m=el_conf:293:9
```

I have been able to replicate this on a [TasteWP](https://tastewp.com/) site with just Site Kit and [GTranslate](https://en-gb.wordpress.org/plugins/gtranslate/) installed. This only displayed when logged into the site as admin and viewing a page with the GTranslate widget place on it. When testing from an incognito browser and not logged in the console error does not display. You do not need to interact with the widget for the console error to show as it will display on page load.

Video recording of this in action:

https://user-images.githubusercontent.com/91896826/194011327-0c30c839-ee09-4140-bfb7-00cc7224306a.mp4

## Steps to reproduce

1. Set up fresh [TasteWP](https://tastewp.com/create/NMS/7.4/5.9.3/google-site-kit/twentytwenty?ni=true&redirect=admin.php%3Fpage%3Dgooglesitekit-splash) website with Site Kit installed and set up.
2. Install the [GTranslate plugin](https://en-gb.wordpress.org/plugins/gtranslate/).
3. Go to **Settings > GTranslate** and select an option for **'Show in menu'** and **'Show floating language selector'** so that the widget will display on posts/pages.
4. Visit the site homepage and console error should display.

## Additional Context

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by asvinb** [comment]
@adamdunnage Weirdly i can reproduce the issue intermittently. First time I tried, I got the error message in the console. Then after a couple of refresh and not doing anything specific that would alter the plugin, the error was not there. Check the screencast below, where I refresh the page, but there are no errors:

https://user-images.githubusercontent.com/31818/199262572-883026e0-0373-4895-b101-a58b8bdac70c.mp4

This will require some further investigation @FlicHollis @eclarke1
